src/app.js

```diff
@@ -0,0 +1,139 @@
+import {div} from '@cycle/dom';
+import _ from 'lodash';
+import xs from 'xstream';
+
+// TO DO:
+// X Render Hello World
+// X Render board
+// X Render knight
+// X Move knight to legal squares by clicking
+// - Move knight to legal squares by dragging
+//
+
+function Board () {
+  return  _.range(0, 8).map(() =>
+    _.range(0, 8).map((square) => Square())
+  );
+}
+
+function Square () {
+  return {}
+}
+
+function isBlack (rowIndex, squareIndex) {
+  if (rowIndex % 2 === 0 && squareIndex % 2 !== 0) {
+    return true;
+  } else if (rowIndex % 2 !== 0 && squareIndex % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
I would probably opt to do this in `CSS`, I think the code reads a bit better:

```js
.row:nth-child(odd) .square:nth-child(even) {
  background: black;
}

.row:nth-child(even) .square:nth-child(odd) {
  background: black;
}
```

My general rule of thumb is to put styles that change dynamically in the code and static styles in my CSS file. That's just what works for me.